### ISSUE TITLE
Custom Lock Card

### ISSUE THREAD

**Issue by hAMMERlized** [neilimixamo/Home-Assistant-Quick-Look-Mobile#14]
Hi,

a lock card would be also a useful addition.

Examples:

- https://github.com/UI-Lovelace-Minimalist/UI/tree/main/custom_cards/custom_card_eraycetinay_lock
- https://github.com/UI-Lovelace-Minimalist/UI/tree/main/custom_cards/custom_card_nik_door

**Comment by neilimixamo** [comment]
Absolutely! Lock cards will enhance the security section, even more with the new Lock dialog that Home Assistant 2023.7 is bringing. I'll provide both simple and expandable versions, ensuring flexibility while maintaining the overall security style, like this :

![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/43101688/e492033f-4b13-4e1c-951b-3c71aee47218)

**Comment by hAMMERlized** [comment]
Nice. Can't wait to see the new improvements!

I think some kind of security step would be useful to prevent opening the door with only one accidental tap.

Something like in the picture below. There is a little lock symbol in the upper right which means the card is blocked for "one tap" actions. You need to double tap the unlock icon on the left than the lock icon on the upper right will disappear for X seconds / minutes and you are good to go. This is a really smart improvement.

And a little icon for the battery status is also useful to see the if a change / reload is needed. Maybe even in 3 steps and colors (75% - 100% = green, 26% - 74% = orange, <=25% = red). What do you think?

![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/a4b52673-1cd0-4ac1-aa85-940b243638a4)

**Comment by neilimixamo** [comment]
Nice, I'll add all these features then. But since I don't own any lock entity, could you please send me a screenshot of one of yours to see which attributes exist (battery, door, ...)

**Comment by hAMMERlized** [comment]
Sure. Here you go. Or do you need all the entities as seperate?

![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/fbac206e-23a8-47c8-a86a-efc969e58f8b)

![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/504625ab-8cd1-4074-b387-954bb336ab93)

**Comment by neilimixamo** [comment]
Thanks, could you also show its attributes, visible in the "Dev/state" panel ? 

![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/43101688/edc78aba-6c0f-4bbe-adf1-86799cccb43c)

**Comment by hAMMERlized** [comment]
Not sure if this helps you as there is not much to see. This is the repo I am using for my nuki door lock [klick](https://github.com/kvj/hass_nuki_ng)

![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/16eabbb5-e421-4e38-a05e-b6471a5d47f7)
![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/da86c863-68a5-4120-aad2-00702d5f2cfa)
![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/bd88b821-bc31-4c3d-9093-ea65a803db77)
![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/0212fe8f-6dfe-4388-b14c-78f66701f8dd)
![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/0b6ea7a1-a28f-4364-98fe-7aefeb59848a)
![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/aee149fa-3bce-4e08-b000-6cae0411c5a9)
![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/aa35348a-0735-43db-87d1-a6a438e5f67c)
![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/6a077def-1e93-4084-82ef-ed64acfbc9a4)
![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/5e15bfaa-900d-4886-98dd-6457b7374e99)

**Comment by neilimixamo** [comment]
Thanks for the screenshots.
Did you create a lock template ? 
I was expecting a `lock.` entity_id instead of a `sensor.` one

**Comment by hAMMERlized** [comment]
Yea, lol. Forgot that one, sorry. Also not much to see 🤷🏻‍♂️

![image](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/assets/95163968/0413f84d-0954-41e3-897f-b8ec8c9152d6)

**Comment by neilimixamo** [comment]
Great, even without many attributes, it still provides me with the understanding I need to manage this entity effectively.

**Comment by neilimixamo** [comment]
Hi, are you happy with the implementation of the new "security v2.0" card, which includes the "lock" functionality? It can also handle contact, occupancy binary_sensors separately or simultaneously, integrates the battery feature, provides attention when in "unlocked" state only and avoids unintended activation by requiring the user to open the (newly introduced 2023.7) lock dialog, wich, I think, keep this card minimalist and consistent with the other cards.

**Comment by hAMMERlized** [comment]
Hey man,

i am sorry but i am pretty buisy with some real life stuff so i can not really test the new release. 

But could you provide me `quick_look_mobile.yaml` 1.0 file please? I have somehow overwritten it and can not use my dashboard anymore. As i have it customized a lot i would have to start from 0 again. Thanks.

**Comment by neilimixamo** [comment]
Yes, unfortunately, the v2 release brings some breaking changes, which is why I recommend keeping the v1 version and installing the new v2 dashboard alongside it to ensure a smoother transition.

<details>
<summary>/config/dashboards/dashboards.yaml</summary>

``` /config/dashboards/dashboards.yaml
quick-look-mobile-dashboard_v1:
  mode: yaml
  title: QLM v1
  icon: mdi:cellphone
  show_in_sidebar: true
  filename: dashboards/quick_look_mobile_v1/qlm.yaml

quick-look-mobile-dashboard:
  mode: yaml
  title: QLM v2
  icon: mdi:cellphone
  show_in_sidebar: true
  filename: dashboards/quick_look_mobile/qlm.yaml
```

</details>

v1 files : [Quick Look Mobile v1.zip](https://github.com/neilimixamo/Home-Assistant-Quick-Look-Mobile/files/11988781/Quick.Look.Mobile.v1.zip)

**Comment by neilimixamo** [comment]
Hi @hAMMERlized, can we consider your issue resolved to close it ?
